app/views/events/_table_badges.html.erb

```diff
@@ -1,30 +1,253 @@
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_pregen" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Pregenerated Characters Only</h4>
+          </div>
+          <div class="modal-body">
+            <p>This scenario may only be played with pregenerated characters. You may not play this with your own
+              character.</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
+
+
 <% end %>
 <% if table.scenario.evergreen? %>
-    <span class="badge badge-evergreen">Evergreen</span>
+    <button type="button" class="btn btn-round-xs btn-evergreen btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_evergreen">Evergreen</button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_evergreen" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Evergreen Scenario</h4>
+          </div>
+          <div class="modal-body">
+            <p>This scenario may be replayed on multiple characters for credit. See the Pathfinder Society Roleplaying
+              Guild Guide (or similar document for Starfinder, etc)</p>
+          </div>
+          <div class="modal-footer">
+            <button type="button" class="btn btn-default" data-dismiss="modal">Close</button>
+          </div>
+        </div>
+      </div>
+    </div>
 <% end %>
+
 <% if table.scenario.hard_mode? %>
-    <span class="badge badge-hardmode"><small>Hard Mode</small></span>
+    <button type="button" class="btn btn-round-xs btn-hardmode btn-xs" data-toggle="modal" data-target="#table_<%= table.id %>_hard">Hard
+      Mode
+    </button>
+    <!-- Modal -->
+    <div id="table_<%= table.id %>_hard" class="modal fade" role="dialog">
+      <div class="modal-dialog">
+        <!-- Modal content-->
+        <div class="modal-content">
+          <div class="modal-header">
+            <button type="button" class="close" data-dismiss="modal">&times;</button>
+            <h4 class="modal-title">Hard Mode Exists</h4>
+          </div>
+          <div class="modal-body">
+            <p>This scenario may played in a special "Hard Mode" where the risk is increased, but for no extra reward.
+              The only extra reward is being able to say you did it in Hard Mode.</p>
+          </div>
```

Comment:
Reword: This scenario has an optional "Hard Mode" for extra challenge (but no additional reward other than bragging rights).